go-controller/pkg/cni/types.go

```diff
@@ -166,6 +166,9 @@ type PodRequest struct {
 
 	// the DeviceInfo struct
 	deviceInfo nadapi.DeviceInfo
+
+	// indicates if interface is primaryUDN
+	IsPrimaryUDN bool
```

Comment:
@qinqon is this boolean really needed? isn't there another we used to infer if its a pudn pod or not? isn't that how we plug two interfaces at the same time?